qa/rpc-tests/zcjoinsplit.py

```diff
@@ -36,6 +36,14 @@ def run_test(self):
         receive_result = self.nodes[0].zcrawreceive(zcsecretkey, joinsplit_result["encryptednote1"])
         assert_equal(receive_result["exists"], True)
 
+        # The pure joinsplit we create should be mined in the next block
+        # despite other transactions being in the mempool.
+        addrtest = self.nodes[0].getnewaddress()
+        for xx in range(0,10):
+            self.nodes[0].generate(1)
+            for x in range(0,50):
+                self.nodes[0].sendtoaddress(addrtest, 0.01);
+
```

Comment:
I debugged the mempool at this point and, as expected, there were 50 txs due to the last iteration of the outer loop..  The mining of 10 blocks seems unnecessary.  You could just have the 'for x' loop to create transactions, whilst dropping the 'for xx' outer loop.